projects/concurrency/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+type Cache struct {
+	capacity int
+	storage  map[int]entry
+}
+
+type entry struct {
+	value    any
```

Comment:
What are the differences between using `any` here vs using [generics](https://gobyexample.com/generics)?